src/rgw/rgw_sal_motr.cc

```diff
@@ -528,18 +529,20 @@ std::unique_ptr<Object> MotrBucket::get_object(const rgw_obj_key& k)
 int MotrBucket::list(const DoutPrefixProvider *dpp, ListParams& params, int max, ListResults& results, optional_yield y)
 {
   int rc;
-  vector<string> key_vec(max);
-  vector<bufferlist> val_vec(max);
+  vector<string> keys(max);
+  vector<bufferlist> vals(max);
```

Comment:
I try to type buffer::list (or ceph::buffer::list) for this; I know, most people don't seem to

Reply:
`bufferlist` is less to type :)

Reply:
that must be it